### PR TITLE
Adding operating kind support to local predictions

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

**Comment by mmerce** [comment]
Additional tests for operating_kind missing. Please don't merge yet.

File: bigml/bigmlconnection.py [hunk 1]
```diff
@@ -165,9 +167,10 @@ def __init__(self, username=None, api_key=None, dev_mode=False,
         values of the `BIGML_USERNAME` and `BIGML_API_KEY` environment
         variables respectively.
 
-        If `dev_mode` is set to `True`, the API will be used in development
-        mode where the size of your datasets are limited but you are not
-        charged any credits.
+        dev_mode` has been deprecated. Now all resources coexist in the
+        same production environment. Existing resources generated in
+        development mode have been archived under an special project and
```
Comment (jaor) [inline bigml/bigmlconnection.py]: typo: a special

File: bigml/bigmlconnection.py [hunk 2]
```diff
@@ -226,7 +232,15 @@ def __init__(self, username=None, api_key=None, dev_mode=False,
     def _set_api_urls(self, dev_mode=False, domain=None):
         """Sets the urls that point to the REST api methods for each resource
 
+        dev_mode` has been deprecated. Now all resources coexist in the
+        same production environment. Existing resources generated in
+        development mode have been archived under an special project and
```
Comment (jaor) [inline bigml/bigmlconnection.py]: same typo

File: docs/101_cluster.rst [hunk 3]
```diff
@@ -92,6 +92,21 @@ class in the `cluster` module. A simple example of that is:
     local_cluster.centroid({"petal length": 4, "sepal length": 2,
                             "petal width": 1, "sepal witdh": 3})
 
+And if you want find out locally the associated centroids
```
Comment (jaor) [inline docs/101_cluster.rst]: typo: to find out

File: docs/index.rst [hunk 4]
```diff
@@ -5534,18 +5579,34 @@ only the prediction will be obtained when applied to regressions.
...
+is used to predict, each category has a confidence, a
+probability or a vote associated to this prediction.
+Then, through the collection
+of models in the
+ensemble, each category gets an averaged confidence, probabiity and number of
+votes. Thus you can decide whether to operate the ensemble using the
+``confidence``, the ``probability`` or the ``votes`` so that the predicted
+category is the one that scores higher in any of these quantities. The
+criteria can be set using the `operating_kind` option (default is set to
+``probabiity``):
```
Comment (jaor) [inline docs/index.rst]: typo: probability

**Pull request merged**